hardware_interface/test/interface_manager_test.cpp

```diff
@@ -84,6 +84,93 @@ TEST(InterfaceManagerTest, InterfaceRewriting)
   EXPECT_EQ(2, foo_iface_ptr->foo);
 }
 
+struct TestCombinationHandle {
```

Comment:
Nit: I think these two struct/classes should be above all the tests (ie. ln 50). Group all the type definitions, and group all the tests.

Reply:
Applied in rebased commits